systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -304,4 +306,36 @@ public void testDeployKafkaOnPersistentStorage() {
         assertThat(configMap, valueOfCmEquals("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "5"));
         assertThat(configMap, valueOfCmEquals("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "5"));
     }
-}
+
+    @Test
+    @KafkaCluster(name = "my-cluster", kafkaNodes = 1, zkNodes = 2, config = {
+            @CmData(key = "zookeeper-healthcheck-delay", value = "30"),
+            @CmData(key = "zookeeper-healthcheck-timeout", value = "10"),
+            @CmData(key = "kafka-healthcheck-delay", value = "30"),
+            @CmData(key = "kafka-healthcheck-timeout", value = "10"),
+            @CmData(key = "KAFKA_DEFAULT_REPLICATION_FACTOR", value = "2"),
+            @CmData(key = "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", value = "5"),
+            @CmData(key = "KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", value = "5")
+    })
+    @OpenShiftOnly
+    public void testForUpdateValuesInConfigMap() {
+        String clusterName = "my-cluster";
+        int expectedZKPods = 2;
+        Oc oc = (Oc) this.kubeClient;
+
+        replaceCm(clusterName, "zookeeper-healthcheck-delay", "23");
+        oc.waitForStatefulSet(zookeeperStatefulSetName(clusterName), expectedZKPods);
+
+        String configMap = kubeClient.get("cm", clusterName);
+        assertThat(configMap, valueOfCmEquals("zookeeper-healthcheck-delay", "23"));
```

Comment:
Are you going to check the updating of some other variables like: 
_"zookeeper-healthcheck-timeout", 
"kafka-healthcheck-delay", 
"kafka-healthcheck-timeout", 
"KAFKA_DEFAULT_REPLICATION_FACTOR", 
"KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR"_?